### PR TITLE
Add Asteroids to the whitelist. Remove Collatz and Cosmology. Fix order

### PR DESCRIPTION
**Pull request by barton2526**
- Asteroids is back
- Collatz and Cosmology have been dead for a while
- SiDock comes before SRBase

### CODE REVIEW

File: _data/whitelist.yml [hunk 1]
```diff
@@ -11,6 +11,20 @@ projects:
...
+    asteroid together with the rotation period and the direction of the spin axis
+    are derived.
+  sponsor: Charles University in Prague
+  cpu: 'yes'
+  gpu: 'yes'
+  gdpr: 'no'
+  team: https://asteroidsathome.net/boinc/team_display.php?teamid=2218
+  stats: https://gridcoinstats.eu/project/asteroids@home
+
 - name: Collatz Conjecture
```
Comment (kjreills) [inline _data/whitelist.yml:28]: Should we also remove some of the other greylisted projects while we're at it? Or do we want a separate PR for making it possible to display projects as greylisted in the page?

Comment (barton2526) [reply _data/whitelist.yml:28]: I suppose we could remove Collatz. Any other projects that are basically dead and on the list?

Comment (barton2526) [reply _data/whitelist.yml:28]: Cosmology?

**Review by jamescowens — APPROVED** [review]
Looks good to me...
